create/mysql.go

```diff
@@ -0,0 +1,98 @@
+package create
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	infra "github.com/ninech/apis/infrastructure/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+)
+
+type mySQLCmd struct {
+	Name                  string                                 `arg:"" default:"" help:"Name of the MySQL instance. A random name is generated if omitted."`
+	Location              string                                 `default:"nine-cz41" help:"Location where the MySQL instance is created."`
```

Comment:
Not really a big fan of the "variable interpolation" tbh. Looks very intransparent for me to find out where the vars come from.

Reply:
Yeah, I know this is not really obvious and a simple `Help() string` method here would do the job as well. The advantage of `enums` is the validation which you get for free, but as this will happen on the API side too, we might just skip it here.